include/ccf/actors.h

```diff
@@ -31,6 +32,10 @@ namespace ccf
       {
         return "node";
       }
+      case ActorsType::acme_challenge:
```

Comment:
We may decide to add more endpoints under `.well_known/` in the future, so perhaps the actors type should be `well_known` to cater for future use cases?

Reply:
Yeah, I think so too. That's why I would have liked a two-level actor prefix.